cmd/shortener/main.go

```diff
@@ -96,6 +101,24 @@ func run() error {
 	router.Get(baseURL.Path+"/{id}", logger.RequestLogger(app.Redirect))
 	router.Get("/api/user/urls", logger.RequestLogger(authhandler.AuthHandle(app.UserUrls)))
 	router.Delete("/api/user/urls", logger.RequestLogger(authcookiehandler.AuthCookieHandle(app.DeleteUserUrls)))
+	router.Get("/api/internal/stats", logger.RequestLogger(app.InternalStats))
+
+	grpcServer := grpc.NewServer()
+	proto.RegisterShortenerServiceServer(grpcServer, &gRPCHandlers.GRPCServer{Storage: storage})
+
+	go func() {
+		lis, err := net.Listen("tcp", config.FlagRunAddr)
+		if err != nil {
+			fmt.Println("failed to listen:", err)
+			os.Exit(1)
```

Comment:
```suggestion
			log.Fatal("failed to listen:", err)
```
fmt.Println на последних итерациях проекта - это моветон.

Reply:
Внес исправление